androidenhancedvideoplayer/src/main/java/com/profusion/androidenhancedvideoplayer/components/playerOverlay/Settings.kt

```diff
@@ -42,9 +47,13 @@ fun Settings(
     onDismissRequest: () -> Unit,
     onSpeedSelected: (Float) -> Unit,
     onIsLoopEnabledSelected: (Boolean) -> Unit,
+    selectedQualityTrack: TrackQualityItem,
+    qualityTracks: () -> List<TrackQualityItem>,
+    onQualityChanged: (TrackQualityItem) -> Unit,
     customization: SettingsControlsCustomization
 ) {
     val sheetState = rememberModalBottomSheetState(skipPartiallyExpanded = true)
+    val enableQualitySelector by remember { derivedStateOf { qualityTracks().size > 2 } }
```

Comment:
`derivedStateOf` is used when the variable depends on another `state`. It helps to "filter" the compositions.
In this case `qualityTracks().size` is a parameter and so this function will recompose every time that it changes. Since `qualityTracks().size` isn't a state, in this case is suitable just use `rememeber`.
```
val enableQualitySelector by remember(qualityTracks().size) { qualityTracks().size > 2 }
```

Reply:
Done!